aips/aip-50.md

```diff
@@ -0,0 +1,72 @@
+---
+aip: 50
+title: Change commission rates in delegation pools
+author: junkil-park, michelle-aptos, movekevin, wintertoro
+discussions-to: https://github.com/aptos-foundation/AIPs/issues/249
+Status: In Review
+type: Standard Framework
+created: 09/28/2023
+---
+
+# AIP-50 - Change commission rates in delegation pools
+
+## Summary
+Currently, the commission rate is set by the pool owner at the initiation of the pool and fixed afterward. We propose allowing the pool owner to change the commission rate after delegation pool creation.
+
+There is a risk that the pool owner can change the commission rate without the staker knowing. To mitigate this, the maximum commission rate change per lockup period is capped and must be executed at least 7 days before the lockup cycle ends. The change does not take effect until the next lockup period, so the staker can unstake before the change takes effect. Wallets and staking UIs should listen to update commission events to build notifications to alert stakers.
+
+### Goals
+This feature will increase the flexibility of pool owners over network/commission rates. Pool owners can adjust operator commission rates as appropriate, based on market conditions and commercial discussions. As such, this will facilitate staking pools to participate in market dynamics. I.e. the pool can be set at a lower commission to attract more stakers, or raise commission when needed. Operators can call the function themselves if they are also the pool owner, or if not, work with the pool owner they are operating for.
+
+### Out of Scope
+Operators will not be able to change the commission rate. In our existing design setup, operators are designated by pool owners.
+
+### Impact
+* *Delegation pool owners* have permission to change the operator commission rate.
+* *Operators* are impacted by commission rate changes, which will impact their rewards.
+* *Delegators* need to watch over commission rate changes so that they are not overcharged. They may assume that the commission rate is fixed.
+
+## Alternative solutions
+There is no alternative solution because the delegation pool module currently does not allow changing the operator commission rate.
+
+Moreover, we decided against allowing operators to change the commission rate, as it would not make sense in the context of our existing design setup. Right now, Operators are designated by pool owners, so eventually, pool owners control everything.
```

Comment:
nit: Rephrase "we decided against" to "allowing operators to change the commission rate would not make sense ..."

Reply:
Done fixing it!